### PR TITLE
Header/Footer - Pattern Library V1.5.2

### PR DESCRIPTION
**Pull request by boxadesign**
#### Header
* Updated to the latest design from Pattern Library release v1.5.0.
* Updated NI and Census theme files
* Removed header2.html file and references and updated to use just header.html

#### Footer
* Updated to the latest design from Pattern Library release v1.5.0.
* NI and Census theme footer2.html file has not been updated with latest markup. On `master` both themes use the main/standard footer and not the footer2.html files in their respective theme folders so this has remained the same.

#### Icons
* Guidance and details components have been updated so that they reference the new icon approach and display correctly.

### How to review 
Check the above components across business, social, census and NI surveys. 

### Checklist

* [ ] Header displays correctly across all themes/survey types.
* [ ] Footer displays correctly across all themes/survey types.
* [ ] Guidance and details components display and function correctly.
* [ ] Code clean up has removed the correct files/code.

### CODE REVIEW

File: app/templates/introduction.html
Comment (insacuri) [inline app/templates/introduction.html]: Need to remove all the random space from all the re-added feedback sections

**Review by insacuri — CHANGES_REQUESTED** [review]
random spaces in templates needs removing

File: app/templates/layouts/_base.html
Comment (insacuri) [inline app/templates/layouts/_base.html]: extra space here

File: app/assets/styles/themes/northernireland/components/_logo.scss
Comment (insacuri) [inline app/assets/styles/themes/northernireland/components/_logo.scss]: very minor, but does that extra line need to be there?

File: app/templates/layouts/_questionnaire.html
Comment (insacuri) [inline app/templates/layouts/_questionnaire.html]: same with this, does it need to be added?

**Review by insacuri — APPROVED** [review]
Other then those 2 minor comments (that aren't critical) looks good

**Review by MebinAbraham — COMMENTED** [review]
We shouldn't merge this until all breaking changes relating to runner are addressed (accordians/radio buttons etc) and ready to go in.

Also padding issue:
screen shot 2018-08-07 at 12 28 23

**Review by insacuri — CHANGES_REQUESTED** [review]
Didn't notice the padding change, needs to be fixed first!

**Comment by boxadesign** [comment]
@MebinAbraham Can you provide the page where you saw the padding issue. I had already addressed but must have missed a template.

**Comment by MebinAbraham** [comment]
For other's reference, it was summary pages.
👍

**Comment by benhowes** [comment]
I see this on questions with radio button answers with the latest commit (since we're on buttons in this thread): 

screenshot from 2018-08-07 15-34-46

**Comment by boxadesign** [comment]
@benhowes As discussed, unable to replicate locally.

**Comment by benhowes** [comment]
@boxadesign Seems that it was caching - I've not been able to see it again after force refreshing!

**Comment by benhowes** [comment]
Looks great! I haven't found anything to compare the census headers against though

**Comment by benhowes** [comment]
Appears to resolve https://github.com/ONSdigital/eq-survey-runner/issues/1505 too, so that's a bonus :)

**Comment by boxadesign** [comment]
@benhowes I have a PR for #1103 which will be approved after lunch. I'll push a new release of the patter library (v1.5.1) and update runner.

**Comment by boxadesign** [comment]
I have updated with v1.5.1 of the pattern library which fixes the text rendering issue of FF.

**Comment by insacuri** [comment]
Did anyone else have any trouble with the javascript on the page?  I'm running it using eq-compose and the accordion bits on the introduction pages aren't working and throwing:
`bundle.min.js:10266 Uncaught TypeError: Cannot read property 'getAttribute' of undefined` when I click on the link.

**Comment by boxadesign** [comment]
@insacuri This is because the mark up for the accordion is incorrect for the latest pattern library release. The correct mark up has been added for this branch - > https://github.com/ONSdigital/eq-survey-runner/pull/1720

**Review by insacuri — APPROVED** [review]
Tested 5 different surveys and everything looks as I would expect

**Review by dcdarrell9 — APPROVED** [review]

**Comment by insacuri** [comment]
Just added do not merge label until the matching work in RAS has been completed.

**Review by MebinAbraham — CHANGES_REQUESTED** [review]
No padding bottom for button?

***Before:***
screen shot 2018-08-22 at 10 44 34
***After:***
screen shot 2018-08-22 at 10 44 24

This is on the screen after submission.

File: app/templates/partials/questions/repeatinganswer.html
Comment (MebinAbraham) [inline app/templates/partials/questions/repeatinganswer.html]: Not related to this PR. We need to mark all static content like this for translation. Feel free mark like the above or leave it.

**Comment by jonnyshaw89** [comment]
@MebinAbraham are you still blocking this?

**Review by MebinAbraham — APPROVED** [review]

**Review by MebinAbraham — CHANGES_REQUESTED** [review]
Actually, it turns out the `Print this page` buttons on view submission no longer works as expected.

**Comment by MebinAbraham** [comment]
I've added the change to make the print button. Check it does work in fact work.

**Comment by MebinAbraham** [comment]
Also updated the translation file.

**Comment by MebinAbraham** [comment]
@boxadesign Just seen you haven't updated the NI theme files. I am pretty sure that you do need to update this. They ***are*** using the footer files within the respective theme files, not the main directory.

File: app/templates/layouts/_base.html
Comment (MebinAbraham) [inline app/templates/layouts/_base.html]: This empty line can be removed.

Comment (boxadesign) [reply app/templates/layouts/_base.html]: Done

**Comment by boxadesign** [comment]
@MebinAbraham I have added in the theme footers and also added the transactional variant of the footer across all themes.

**Comment by MebinAbraham** [comment]
@boxadesign I can't see the new changes?. Did you push?.

**Comment by boxadesign** [comment]
@MebinAbraham Should be good to go now.

**Review by MebinAbraham — APPROVED** [review]
https://github.com/ONSdigital/eq-survey-runner/blob/3652978f03dbbbe7589f8fd42d6fe7eaccb4eedb/app/assets/styles/partials/components/_footer2.scss

Is everything from file above still needed?. It references `footer2`?

The translations `messages.pot` needs updating also.

File: app/templates/layouts/_base.html
Comment (MebinAbraham) [reply app/templates/layouts/_base.html]: Still there.

**Comment by boxadesign** [comment]
@MebinAbraham I have removed that file at it wasn't being used anywhere.

**Review by MebinAbraham — COMMENTED** [review]
@boxadesign Quick change. Not sure if you are aware but the Star Wars theme doesn't display correctly, we do not care much for this theme but it seems to be just because of reference error.

For Star Wars, the `header.html` is actually called `topbar.html` hence the broken layout. Can you rename this to `header.html` which fixes the issue.

File: app/templates/partials/question-definition.html
Comment (MebinAbraham) [inline app/templates/partials/question-definition.html]: `Click to expand` needs to be marked for translation like before. Need to update `messages.pot` after.

**Review by MebinAbraham — CHANGES_REQUESTED** [review]
Just those 2 changes to make. Can also squash this when you push as everything seems to be as expected 👍

**Comment by boxadesign** [comment]
@MebinAbraham Those changes are now in to check

File: app/templates/partials/question-definition.html
Comment (MebinAbraham) [inline app/templates/partials/question-definition.html]: @boxadesign Very small but need the full stop?. Was there before and is there for `Click to expand.` text above.
Can rebase and squash.

File: app/themes/starwars/templates/layouts/_base.html
Comment (MebinAbraham) [inline app/themes/starwars/templates/layouts/_base.html]: Sorry to be a pain, can we remove these empty lines.
Also if you add `<!-- Theme selected: Star Wars -->` then the failing test will pass.

**Review by MebinAbraham — APPROVED** [review]

**Pull request merged**
